mix.exs

```diff
@@ -118,6 +118,7 @@ defmodule Meadow.MixProject do
         "meadow.ldap.setup test/fixtures/ldap_seed.ldif",
         "ecto.create --quiet",
         "ecto.migrate",
+        "run priv/repo/seeds.exs",
```

Comment:
It's not running for me as part of `mix meadow.setup`. I had to run it manually. Is it supposed to be?

Reply:
Nope, it's definitely supposed to be part of setup. I'll add it in.